towerlib/entities/job.py

```diff
@@ -1649,6 +1826,42 @@ def summary_fields(self):
         """
         return self._data.get('summary_fields')
 
+    @property
+    def related(self):
+        """The related fields of the job.
+
+        Returns:
+            dict: The related fields of the job.
+
+        """
+        return self._data.get('related')
+
+    @property
+    def events(self):
+        """The events of the job.
+
+        Returns:
+            dict: The events of the job.
+
+        """
+        url = self._data.get('related', {}).get('events')
+        return EntityManager(self._tower,
+                             entity_object='JobEvent',
+                             primary_match_field='id',
+                             url=url)
+
+    @property
+    def event_processing_finished(self):
+        """The event_processing_finished field of the job.
+
+        Returns:
+            bool: The event_processing_finished field of the job.
+
+        """
+        url = f'{self._tower.api}/project_updates/{self.id}/'
+        response = self._tower.session.get(url)
+        return response.json().get('event_processing_finished', {}) if response.ok else None
```

Comment:
If this should return a bool, then on the `.get()` the default value should be a bool and not a dictionary. I would also suggest to check for the response.ok because it might be that the response itself failed and in that case you would want to know what went wrong.